packages/liferay-npm-scripts/src/utils/mergeBabelLoaderOptions.js

```diff
@@ -22,30 +22,45 @@ function mergeBabelLoaderOptions(webpackConfig) {
 		return webpackConfig;
 	}
 
-	webpackConfig.module.rules.forEach((rule) => {
-		let {use} = rule;
-
-		if (!use) {
-			return;
-		}
-
-		if (!Array.isArray(use)) {
-			use = [use];
-		}
-
-		use.forEach((useEntry, i) => {
-			if (typeof useEntry === 'string') {
-				use[i] = {
-					loader: useEntry,
-					options: {...BABEL_CONFIG},
+	return {
+		...webpackConfig,
```

Comment:
We might also want to add the ability for multiple webpack configs, meaning it could be an array of webpack configs.